### PR TITLE
add lastparked logic

### PR DESCRIPTION
**Pull request by murveit**
Modify the AstroPhysics V2 driver so that it uses the "LastParked" API logic to recover when mount connects uninitialized.

Also, cleans up the source file--there was a lot of dead code and obsolete comments inherited from previous driver iterations.

### CODE REVIEW

File: drivers/telescope/lx200ap_v2.cpp [hunk 1]
```diff
@@ -990,13 +729,23 @@ char* trackStateString(INDI::Telescope::TelescopeStatus state)
 
 bool LX200AstroPhysicsV2::ReadScopeStatus()
 {
+    if (!isAPReady())
+    {
+        char dbgStr[1000];
+        sprintf(dbgStr, "APStatus: Not ready--Checked %s Initialized %s Time updated %s Location Updated %s",
+                apInitializationChecked ? "Y" : "N", apIsInitialized ? "Y" : "N",
+                apTimeInitialized ? "Y" : "N", apLocationInitialized ? "Y" : "N");
+        LOGF_DEBUG("%s", dbgStr);
```
Comment (knro) [inline drivers/telescope/lx200ap_v2.cpp:738]: why not use LOGF_DEBUG directly?

File: drivers/telescope/lx200ap_v2.cpp [hunk 2]
```diff
@@ -1023,108 +766,40 @@ bool LX200AstroPhysicsV2::ReadScopeStatus()
...
-    }
     char dbgStr[1000];
-    sprintf(dbgStr,
-            "%s %s stime: %s uOffset %4.1f lTime %6.3f %s LL: %03d:%02d %04d:%02d RA/DEC: %.3f %.3f AltAz: %.3f %.3f HA: %.3f",
-            trackStateString(TrackState), isAPParked ? "Parked" : "Unparked", sTimeStr.c_str(),
-            val_utc_offset, lTimeDbl, dateStr.c_str(), latD, latM, lonD, lonM,  currentRA,
-            currentDEC, Alt, Az, ha);
+    sprintf(dbgStr, "APStatus: %s %s stime: %s  RA/DEC: %.3f %.3f",
+            trackStateString(TrackState), apParked ? "Parked" : "Unparked", sTimeStr.c_str(), currentRA, currentDEC);
     LOGF_DEBUG("%s", dbgStr);
```
Comment (knro) [inline drivers/telescope/lx200ap_v2.cpp:802]: same as above.

File: drivers/telescope/lx200ap_v2.cpp [hunk 3]
```diff
@@ -1222,6 +874,7 @@ bool LX200AstroPhysicsV2::ReadScopeStatus()
     syncSideOfPier();
 
     // Hack -- when the ra/dec doen't change, the hour angle info isn't being updated. Not sure how to fix...
+    // This may be unnecessary now with changed to KStars. Hy 3/11/2022.
     EqN[AXIS_RA].value = currentRA;
     EqN[AXIS_DE].value = currentDEC;
     IDSetNumber(&EqNP, nullptr);
```
Comment (knro) [inline drivers/telescope/lx200ap_v2.cpp:880]: Please remove this and use newRADec(...)

**Review by knro — CHANGES_REQUESTED** [review]
Awesome work Hy, AP users are going to love this! Can you bump the version? and also check my other comments.

File: drivers/telescope/lx200ap_v2.cpp (hunk 1 above)
Comment (murveit) [reply drivers/telescope/lx200ap_v2.cpp]: Done.

File: drivers/telescope/lx200ap_v2.cpp (hunk 2 above)
Comment (murveit) [reply drivers/telescope/lx200ap_v2.cpp]: Done (and all the others too).

File: drivers/telescope/lx200ap_v2.cpp (hunk 3 above)
Comment (murveit) [reply drivers/telescope/lx200ap_v2.cpp]: Done. newRaDec() was already there.

**Comment by murveit** [comment]
I upped the version to 1.1, but also changed the name from AstroPhysics Testing to AstroPhysics V2.
I will add in a KStars MR to change its name as well to AstroPhysics V2, as soon as we start 3.5.9.

**Pull request merged**